src/PhpWord/Writer/Word2007/Part/Chart.php

```diff
@@ -312,4 +400,30 @@ private function writeShape(XMLWriter $xmlWriter, $line = false)
         $xmlWriter->endElement(); // a:ln
         $xmlWriter->endElement(); // c:spPr
     }
+
+    private function writeAxisTitle(XMLWriter $xmlWriter, $title)
+    {
+        $xmlWriter->startElement('c:title'); //start c:title
+        $xmlWriter->startElement('c:tx'); //start c:tx
+        $xmlWriter->startElement('c:rich'); // start c:rich
+        $xmlWriter->writeElementBlock('a:bodyPr');
```

Comment:
writeElementBlock expects more attributes, this should be changed to writeElement instead, the same a couple of lines down.

Reply:
more generally, check the (very few) warnings in scrutinizer

Reply:
FYI, if fixed those